inst/components/scss/value_box.scss

```diff
@@ -14,78 +110,141 @@
     padding: 1rem;
     overflow: hidden;
 
-    .bi, .fa {
+    .bi, .fa, .fab, .fas, .far {
       opacity: .85;
       min-width: 50px;
       max-width: 125%;
     }
     // We set font size because {bsicons}/{fontawesome} both
     // set height/width to 1em by default (as an inline style)
     .bi { font-size: 5rem; }
-    .fa { font-size: 4rem; }
+    .fa, .fab, .fas, .far { font-size: 4rem; }
+  }
+
+
+  &.showcase-top-right {
+    .value-box-grid {
+      grid-template-columns: 1fr var(---bslib-value-box-showcase-w, 50%);
 
-    max-height: var(--bslib-value-box-max-height);
+      .value-box-showcase {
+        grid-area: right;
+        margin-left: auto;
+        align-self: start;
+        align-items: end;
+        padding-left: 0;
+        padding-bottom: 0;
+      }
 
-    &.showcase-top-right {
-      align-items: end;
-      padding-left: 0;
-      padding-bottom: 0;
+      .value-box-area {
+        grid-area: left;
+        align-self: end;
+      }
     }
-  }
 
-  // Should also be fillable/fill (i.e., d-flex; flex: 1)
-  .value-box-area {
-    justify-content: center;
-    padding: 1.5rem 1rem;
-    font-size: .9rem;
-    font-weight: 500;
-    * {
-      color: inherit;
-      margin-bottom: 0;
-      margin-top: 0;
+    &[data-full-screen="true"] {
+      .value-box-grid {
+        grid-template-columns: auto var(---bslib-value-box-showcase-w-fs, 1fr);
+        > div {
+          align-self: center;
+        }
+      }
     }
-    // value_box(title = )
-    > :first-child {
-      @include bootstrap-heading($h6-font-size);
-      color: inherit;
-      // add a non-breaking space to ensure it's not 0 height
-      &::after {
-        content: '\00a0 '
+
+    &:not([data-full-screen="true"]) {
+      .value-box-showcase {
+        margin-top: 0;
+        overflow: visible;
```

Comment:
Maybe not needed

```suggestion
```

Reply:
We don't need this anymore, and we also don't need `overflow: hidden` on `.value-box-showcase` either.